src/menu/mod.rs

```diff
@@ -677,17 +676,40 @@ impl<T: std::cmp::PartialEq + std::default::Default + Clone> Menu<T> {
                             );
                             batch.draw(ctx)?;
                         } else {
-                            let mut difficulty_name: String = "Difficulty: ".to_owned();
-
-                            match save.difficulty {
-                                0 => difficulty_name.push_str("Normal"),
-                                2 => difficulty_name.push_str("Easy"),
-                                4 => difficulty_name.push_str("Hard"),
-                                _ => difficulty_name.push_str("(unknown)"),
+                            fn translate_fallback<'a>(result: &'a str, key: &'_ str, fallback: &'a str) -> &'a str {
```

Comment:
I think it would be better if there was no fallback and the normal behavior was used instead (displaying the localize key) because it makes it easier to debug/notice that you missed a key in the locale file. Most of us use English while working on doukutsu-rs, so seeing "Difficulty: menus.difficulty_menu.normal" in the GUI will be more helpful than seeing "Difficulty: Normal" (from the fallback) and blindly assuming we didn't make a mistake.

Reply:
Oki